Engine/media/audio/audio.cpp

```diff
@@ -11,7 +11,8 @@
 // http://www.opensource.org/licenses/artistic-license-2.0.php
 //
 //=============================================================================
-#include <math.h>
+#include <cmath>
+#include <cstdio>
 #include "core/platform.h"
```

Comment:
> `error: cannot initialize a parameter of type 'const wchar_t *' with an rvalue of type 'const char *'`
  when building with Emscripten

This error sounds a little suspicious; do you remember what line does it refer to?

Reply:
Hey, this error is on `audio.cpp:536:20`, The full error message is as following:

```
audio.cpp:536:13: error: use of undeclared identifier 'sscanf'; did you mean 'vswscanf'?
        if (sscanf(game.audioClips[sound_number].scriptName.GetCStr(), "aSound%d", &old_style_number) == 1)
            ^~~~~~
            vswscanf
C:\Users\erico\git\ericoporto\ags\Emscripten\emscripten\emsdk\upstream\emscripten\cache\sysroot/include\wchar.h:123:5: note: 'vswscanf' declared here
int vswscanf (const wchar_t *__restrict, const wchar_t *__restrict, __isoc_va_list);
    ^
C:/Users/erico/git/ericoporto/ags/Engine/media/audio/audio.cpp:536:20: error: cannot initialize a parameter of type 'const wchar_t *' with an rvalue of type 'const char *'
        if (sscanf(game.audioClips[sound_number].scriptName.GetCStr(), "aSound%d", &old_style_number) == 1)
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\Users\erico\git\ericoporto\ags\Emscripten\emscripten\emsdk\upstream\emscripten\cache\sysroot/include\wchar.h:123:40: note: passing argument to parameter here
int vswscanf (const wchar_t *__restrict, const wchar_t *__restrict, __isoc_va_list);
                                       ^
```

Reply:
maybe this and the `Engine: fix some errors from missing cstdio` commit can be picked to a new PR to be applied to current master (see #1434)

Reply:
This was solved by #1475 so I dropped this commit.